### PR TITLE
nuweb: update to 1.62

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/nuweb/Portfile [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-checksums           rmd160  0c32ae064d742ff32bde14ab4a6a269c52772b84 \
-                    sha256  fe909b703c707c04f778c3d3d2f1eed2e10034f7025d292dd9ae041feb88a160
+patchfiles          patch-Makefile.diff
 
 use_configure       no
 
 build.target        nuweb
-build.env-append    CC=${configure.cc} \
-                    "CFLAGS=${configure.cflags} [get_canonical_archflags cc] -std=gnu89 -Wno-return-type"
+build.env-append    "CFLAGS=${configure.cflags} [get_canonical_archflags cc] -std=gnu89 -Wno-return-type"
```
Comment (reneeotten) [inline devel/nuweb/Portfile:27]: `CFLAGS` should also be set by the `makefile` PG as far as I can tell, removing this doesn't work?

Comment (barracuda156) [reply devel/nuweb/Portfile:27]: Well, there are some custom flags added here, I guess there was a reason for these?

Comment (reneeotten) [reply devel/nuweb/Portfile:27]: perhaps for the other ones, but CFLAGS should be set by the `makefile` PG just as `CC`. You only remove the latter, I am asking why not the `CFLAGS`.

Comment (barracuda156) [reply devel/nuweb/Portfile:27]: I will try dropping flags.

Comment (barracuda156) [reply devel/nuweb/Portfile]: OK we are good without flags.

**Pull request merged**